plugins/snowflake/dbt/include/snowflake/macros/catalog.sql

```diff
@@ -24,7 +24,12 @@
               'Approximate Size' as "stats:bytes:label",
               bytes as "stats:bytes:value",
               'Approximate size of the table as reported by Snowflake' as "stats:bytes:description",
-              (bytes is not null) as "stats:bytes:include"
+              (bytes is not null) as "stats:bytes:include",
+
+              'Last Modified' as "stats:last_modified:label",
+              to_varchar(last_altered, 'mon dd, yyyy-HH24:MI:SS') as "stats:last_modified:value",
```

Comment:
I think we should pick between displaying either:
* a concise, correct value: `to_varchar(convert_timezone('UTC', last_altered), 'yyyy-mm-dd HH24:MI'||' UTC')`
* a user-friendly value: `to_varchar(last_altered, 'mon dd, hh12:mi am')`

I lean toward the former, what do you think?